app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,10 @@
+class BookingsController < ApplicationController
+  def create
+    @toilet = Toilet.find(params[:toilet_id])
+    @booking = Booking.new
+    @booking.toilet = @toilet
+    @booking.user = current_user
+    @booking.save
+    redirect_to toilet_path
```

Comment:
add authorize @booking after line 8

Reply:
@RuiSMagalhaes done

Reply:
1)you should put the authorize @booking before the redirect_to
2)line 8 shoud be redirect_to @toilet